.github/workflows/windows.yml

```diff
@@ -243,15 +243,16 @@ jobs:
           . "${{ env.OV_INSTALL_DIR }}/setupvars.ps1"
           python -m pip install ./thirdparty/openvino_tokenizers/[transformers] -r ./tests/python_tests/requirements.txt --find-links ${env:OV_INSTALL_DIR}/wheels
           python -m pytest -v ./tests/python_tests/test_tokenizer.py::test_set_chat_template
+          python -m pytest -v tests/python_tests/test_text_streamer.py
         env:
-          PYTHONPATH: "./build/" # cmd evaluates variables in a different way. Setting PYTHONPATH before setupvars.bat instead of doing that after solves that.
+          PYTHONPATH: "./build/;build/tests/python_tests/pybind_wrapper" # cmd evaluates variables in a different way. Setting PYTHONPATH before setupvars.bat instead of doing that after solves that.
```

Comment:
maybe output binary should also be copied to `./build` to have a single folder with all binaries?

Reply:
Agreed.

Just one question. Won't pybind_wrapper be accidentally copied into the product?

Reply:
no, `/build` is used during development only

Reply:
Since introduced a wheel package with `genai_test_utils`. Answered below.